static_src/test/unit/components/users.spec.jsx

```diff
@@ -1,52 +1,83 @@
 import '../../global_setup.js';
 
 import React from 'react';
+import Immutable from 'immutable';
+import sinon from 'sinon';
 import { shallow } from 'enzyme';
 import Users from '../../../components/users.jsx';
+import PanelDocumentation from '../../../components/panel_documentation.jsx';
+import UsersInvite from '../../../components/users_invite.jsx';
 import UserStore from '../../../stores/user_store';
+import SpaceStore from '../../../stores/space_store';
 
-describe('<Users />', function () {
-  let users, sandbox;
+const buildRoles = (spaceGuid, roles = []) => {
+  const obj = {};
+  obj[spaceGuid] = roles;
 
-  beforeEach(function () {
-    sandbox = sinon.sandbox.create();
-  });
+  return obj;
+};
 
-  afterEach(function () {
-    sandbox.restore();
-  });
+describe('<Users />', () => {
+  const userGuid = 'a-user-guid';
+  const spaceGuid = 'space-guid';
+  const user = {
+    guid: userGuid,
+    roles: buildRoles(spaceGuid, ['org_manager'])
+  };
+
+  let users;
 
-  describe('with a user', function () {
-    beforeEach(function () {
-      const userGuid = 'a-user-guid';
-      const user = {
-        guid: userGuid
-      };
-      UserStore._currentUserGuid = userGuid;
-      UserStore.push(user);
+  SpaceStore._currentSpaceGuid = spaceGuid;
+  UserStore._currentUserGuid = userGuid;
 
+  describe('with a user', () => {
+    beforeEach(() => {
+      UserStore._data = Immutable.fromJS([user]);
       users = shallow(<Users />);
     });
 
-    describe('when at org level', function () {
-      beforeEach(function () {
+    describe('when at org level', () => {
+      it('doesnt have permissions to edit users', () => {
```

Comment:
i know this was here before, but let's change this statement (or change the test, but probably the former). it doesn't test the permissions of the users.